### ISSUE TITLE
undefined or null entry should be supported

### ISSUE THREAD

**Issue by janmisek** [cibernox/ember-power-select#1124]
Hello there 

I need to contain null or undefined in entry list.

Its not currently possible due to following line:
https://github.com/cibernox/ember-power-select/blob/74e5472ae1f821e52286d26c1ffa850f156700d2/addon/components/power-select.js#L520

as ember assertion fail on get(undefined, 'options')

Should this usecase be supported ? or is there any workaround.

**Comment by cibernox** [comment]
Yes, there is a workaround, which is creating wrapper objects like this:
`[ { value: null }, { value: 'one' }, { value: 'two' }, ...]`

I know it's a small inconvenience, but if `null` and `undefined` were supported options of themselves, I'd have to have quite a few null checks everywhere. It's not a one line change. This issue was raised once before but after some tests I decided that it was better to advice people to create wrapper objects for this not-very-common use case.

**Comment by janmisek** [comment]
Ok thanks for explanation
